### DISCUSSION TITLE
lesson10 Error 500

### DISCUSSION THREAD

**Discussion by fhussain** [Cyfrin/foundry-full-course-cu#2540] [Q&A] [answered]
hi! when i am trying to open lesson 10 of smart contract lottery, "The Modulo Operation" at Cyfrin updraft I am getting the error from the website 
error 500
This page must have been forked!

You’ve stumbled onto a path that's no longer supported by our mainnet. It could be an old fork or a sidechain that's gone rogue. Either way, we’ve got a stable mainnet with plenty of validated paths for you to explore.

where can i get the rest of the course?
Thanks.

**Comment by abraj** [comment, marked as answer]
Yes, there seems to be this issue **"Error 500: This page must have been forked!"** on some pages.
I was also getting this error on a few pages some time ago yesterday in **Foundry Fundamentals** course.
However, when I _**refreshed the page**_ a couple of times, then the video loaded successfully.

**Reply by fhussain** [reply]
Thanks it is working!!

**Comment by EngrPips** [comment]
Hello @fhussain, Have you resolved this issue already?

**Reply by fhussain** [reply]
Yes, it is thanks!

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
